Tests/L0/Grunt/invalidTestSource.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I don't think only 2 simple mocks would work because for checking some failures, say for npm, I have to execute gulp/grunt successfully. But for checking gulp/grunt failure, I need to fail execution of gulp/grunt/ though this can't happen in the same file.